components/properties/src/script.rs

```diff
@@ -45,18 +44,98 @@ impl ScriptWithExt {
 }
 
 impl ScriptWithExt {
+    /// Returns whether the [`ScriptWithExt`] value has Script_Extensions and
+    /// also indicates a Script value of [`Script::Common`].
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use icu_properties::script::ScriptWithExt;
+    ///
+    /// assert!(ScriptWithExt(0x04FF).is_common());
```

Comment:
thought: should we use `'ӿ'.into()` instead (perhaps with the codepoint in a comment) for readability?

Reply:
Good question. I wrote these input values in hex notation, but they are not code points. Rather, they are examples of `ScriptWithExt` values in the CPT, which are [that special twist on the `Script` "enum discriminant" values](https://github.com/unicode-org/icu4x/blob/main/components/properties/src/script.rs#L85-L98) -- we start out with a `Script`'s u16 and then twiddle 2 higher-order bits when the code point has Script_Extensions. Since these helper methods are doing bit operations, it made sense to keep the values in hex notation.  

And then none of this really should be user-facing anyways, as Shane's comment points out.

Reply:
Seems fair!